apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -26,7 +26,8 @@ import (
 	"github.com/juju/juju/controller/modelmanager"
 	"github.com/juju/juju/core/life"
 	"github.com/juju/juju/core/permission"
-	modelmanagerservice "github.com/juju/juju/domain/modelmanager/service"
+	"github.com/juju/juju/domain/model"
+	modeldomain "github.com/juju/juju/domain/model"
```

Comment:
This is confusing. `modeldomain` is an alias for the same package imported as `model`.